CIP-0069/README.md

```diff
@@ -41,12 +41,29 @@ This primarily manifests in the ability to use a single validator for both minti
 All the script purposes have a form of `Redeemer -> ScriptContext -> a` except the `Spending` one.
 It has the following form: `Datum -> Redeemer -> ScriptContext -> a`. This is enforced by the Cardano ledger.
 
-We modify the general signature to be `ScriptArgs -> ScriptContext -> a`.
+We make the following modification:
+
+- The signature of all scripts will be `Redeemer -> ScriptContext -> a`.
+- The `ScriptPurpose` type is modified such that the `Spending` variant contains a `Datum`:
+
+```hs
+data ScriptPurpose
+  = ...
+  | Spending TxOutRef Datum
```

Comment:
Should this be Maybe Datum instead of Datum? Or is this a Datum field containing
Datum Hash, InlineDatum, No Datum?